sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -233,6 +233,10 @@ export class ApplicationStudentsController extends BaseController {
       payload.data.selectedOfferingEndDate = studyEndDate;
     }
 
+    // Set studentDataAdditionalTransportKm to 0 for Camunda assessment calculation
+    // if it is not provided from the student application payload
+    payload.data.studentDataAdditionalTransportKm =
+      payload.data.studentDataAdditionalTransportKm ?? 0;
```

Comment:
👍

Reply:
The dynamic data is now handled in the workflow level.
https://github.com/bcgov/SIMS/pull/2824#discussion_r1491836799